README.md

```diff
@@ -1,4 +1,23 @@
+This program will call the hcship.DestroyLayer system call on the specified folder
 
-run file:
+This will effectively reclaim a lot of hard-drive space for an undocumented reason.
 
-.\docker-ci-zap.exe -folder "C:\ProgramData\docker"
+# Usage (in admin PS console)
+
+1) Stop docker
+
+```powershell
+    net stop docker
+```
+2) run file:
+
+```powershell
+    cd "C:\ProgramData\docker"
+    .\docker-ci-zap.exe -folder "."
+```
+
+3) restart docker
+
+```powershell
+    net start docker
+```
```

Comment:
It may well be necessary to reboot between 1 and 2 to ensure there are no containers running which have ref counts to files which cannot be removed.